src/span.js

```diff
@@ -106,6 +122,9 @@ export default class Span {
             if (this.getBaggageItem(normalizedKey)) {
                 logs['override'] = 'true';
             }
+            if (truncated) {
+                logs['truncated'] = 'true';
```

Comment:
Is there a better way of capturing this data? 
With multiple baggage items, it might be difficult to figure out which one was truncated